include/functions.h

```diff
@@ -871,47 +871,40 @@ void Font_LoadChar(Font* font, u8 character, u16 codePointIndex);
 void Font_LoadMessageBoxEndIcon(Font* font, u16 icon);
 void Font_LoadOrderedFont(Font* font);
 s32 func_8006F0A0(s32 arg0);
-// ? func_8006F0D4(?);
-// ? func_8006F0FC(?);
-u8 func_8006F140(GlobalContext*, EnvironmentContext*, UNK_TYPE);
-f32 func_8006F93C(u16, u16, u16);
-f32 func_8006F9BC(u16 endFrame, u16 startFrame, u16 frames, u16, u16);
-// ? func_8006FB94(?);
-void func_8006FC88(u8 skyboxId, EnvironmentContext* envCtx, SkyboxContext* skyboxCtx);
-void func_80070600(GlobalContext* globalCtx, s32 waterboxProperties);
-void func_800706A0(GlobalContext* globalCtx);
-// ? func_80070718(?);
-void func_80070C24(GlobalContext* globalCtx, EnvironmentContext* envCtx, LightContext* lightCtx, PauseContext* pauseCtx,
+u16 Kankyo_GetPixelDepth(s32 x, s32 y);
+void Kankyo_GraphCallback(GraphicsContext* gfxCtx, void* param);
+void Kankyo_Init(GlobalContext* globalCtx, EnvironmentContext* envCtx, s32 unused);
+f32 Kankyo_LerpWeight(u16, u16, u16);
```

Comment:
I noticed a lot of the prototypes here aren't updated with all arguments or their names, is it something that's waiting for the future documentation pass?